numpy/distutils/ccompiler.py

```diff
@@ -7,9 +7,10 @@
 import subprocess
 from copy import copy
 from distutils import ccompiler
-from distutils.ccompiler import *
+from distutils.ccompiler import compiler_class, gen_lib_options, \
+    get_default_compiler, new_compiler, CCompiler
```

Comment:
Can you use parens instead of backslashes here?

Reply:
done